maintenance_request_time_performance/data/maintenance_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data>
+
+    <!-- Standard stages: default in progress stage -->
+    <function
+            id="function_default_stage_in_progress"
+            model="maintenance.stage"
+            name="update_default_stage_in_progress"/>
+
+</data>
```

Comment:
I think this can be achieved in a better way (locally tested):

```suggestion
    <data noupdate="1">
        <record id="maintenance.stage_1" model="maintenance.stage">
            <field name="in_progress" eval="True" />
        </record>
    </data>
```